streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/master/JobMaster.java

```diff
@@ -243,6 +247,29 @@ public byte[] requestJobWorkerRollback(byte[] requestBytes) {
     return RemoteCall.BoolResult.newBuilder().setBoolRes(ret).build().toByteArray();
   }
 
+  public Boolean destroy() {
+    LOG.info("Start to destroy job master for job: {}.", runtimeContext.getJobName());
+    long startTs = System.currentTimeMillis();
+
+    if (scheduler != null) {
+      scheduler.destroyJob();
+    }
+
+    if (checkpointCoordinator != null) {
+      checkpointCoordinator.stop();
+    }
+
+    if (failoverCoordinator != null) {
+      failoverCoordinator.stop();
+    }
+
+    LOG.info(
```

Comment:
Actually it's finished to destory job master here.

Reply:
Keep the same logic with the internal implementations for now.